pkg/snapstore/snapshot.go

```diff
@@ -126,6 +126,10 @@ func ParseSnapshot(snapPath string) (*brtypes.Snapshot, error) {
 	//parse creation time as well as parse the Snapshot compression suffix
 	lastNameToken := strings.Split(tokens[3], "/")
 	timeWithSnapSuffix := strings.Split(lastNameToken[0], ".")
+	// Check & remove if the last token is a chunk directory. The chunkDirSuffix is set by only GCS snapstore when using a emulator for testing.
+	if "."+timeWithSnapSuffix[len(timeWithSnapSuffix)-1] == brtypes.ChunkDirSuffix {
```

Comment:
Please use fmt.Sprintf() to construct the LHS string, since that will be more readable.